modules/admin/src/app/modules/adminPanel/lectors/lectors.component.ts

```diff
@@ -205,15 +275,28 @@ export class LectorsComponent implements OnInit {
 
   formatDate(dateString) {
     if (dateString == '-') {
-      return '-';
+      return '';
     }
 
-    let date = new Date(dateString);
+    let date = new Date(dateString + 'Z');
 
     let year = date.toLocaleDateString('en-US', { year: 'numeric' });
     let month = date.toLocaleDateString('en-US', { month: '2-digit' });
     let day = date.toLocaleDateString('en-US', { day: '2-digit' });
     let time = date.toLocaleTimeString('en-US', { hour12: false });
-    return `${day}/${month}/${year} ${time}`;
+    return `${day}.${month}.${year}, ${time}`;
+  }
+
+  getStatusTooltip(lecturer: Professor) {
+    let tooltip = "";
+    
+    if (lecturer.IsActive) {
+      return;
+    }
+
+    if (lecturer.DeletionDate != null) {
+      return "Удален\r\n" +
```

Comment:
Should we use translation here?

Reply:
There is a front-end developer who will be working on translation of admin module.